src/rshim.c

```diff
@@ -187,11 +188,19 @@ rshim_backend_t *rshim_devs[RSHIM_MAX_DEV];
 char *rshim_dev_names[RSHIM_MAX_DEV];
 char *rshim_blocked_dev_names[RSHIM_MAX_DEV];
 
+/* Bitmask of the used rshim device id. */
+#if RSHIM_MAX_DEV > 64
```

Comment:
DId you add this to fix it in the future?
Since it is set to 64 right now?

Reply:
This commit uses a 64-bit bitmask to identify the connected rshim devices. It's based on the assumption that the RSHIM_MAX_DEV won't be bigger than 64.

This line is a robust check to prevent us increasing RSHIM_MAX_DEV larger than 64 by accident.

(64 should be good enough. If more rshim devices are needed someday, we could split them into different processes).